solosis/commands/helpers/tokens.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+import random
+import string
+
+
+def generate_token(length=10):
```

Comment:
This function isn't being used anywhere at the moment. What is the intended usage?